support/config/deployment.go

```diff
@@ -206,7 +206,6 @@ func (c *DeploymentConfig) ApplyTo(deployment *appsv1.Deployment) {
 	c.Resources.ApplyTo(&deployment.Spec.Template.Spec)
 	c.LivenessProbes.ApplyTo(&deployment.Spec.Template.Spec)
 	c.ReadinessProbes.ApplyTo(&deployment.Spec.Template.Spec)
-	c.Resources.ApplyTo(&deployment.Spec.Template.Spec)
```

Comment:
These seem all unrelated?

Reply:
> These seem all unrelated?

@alvaroaleman , these remove duplication -- yes, unrelated, but not worth opening a separate PR.

Reply:
yes please, I think it does. If I came to a git diff of this PR with no context or in a few months I'm pretty sure I'll expend a valuable amount of time thinking how these changes relates to the PR goal.
As a good practices let's drop the unrelated changes from the PR and open a separate one for removing duplication.